src/lib/utils/evm/index.ts

```diff
@@ -93,9 +91,15 @@ export const isGetFeesWithWalletEIP1559SupportArgs = (
 ): input is GetFeesWithWalletEip1559SupportArgs =>
   Boolean(input.adapter && input.wallet && input.data && input.to && input.from)
 
+export type EvmFees = {
```

Comment:
was there a reason this couldn't stay in the chain-adapters `evm/utils`?

Reply:
yeah it causes a dependency cycle which is why so much was moved around actually

Reply:
went ahead and moved it next to the hook that actually leverages and cares about it (plus some additional import cleanup I missed): https://github.com/shapeshift/web/pull/7612/commits/dd7795e10bbe3c542f43a781c908827050bfacf1